docs/source/conf.py

```diff
@@ -128,10 +128,21 @@
 html_theme_options = {
```

Comment:
Does
```
html_context = {
    "display_github": True, # Integrate GitHub
    "github_user": "stfc", # Username
    "github_repo": "janus-core", # Repo name
    "github_version": "main", # Version
    "conf_py_path": "/docs/", # Path in the checkout to the docs root
}
```
work?

Reply:
I don't think it adds a GitHub link anywhere, if that's what you mean.

I was also hoping to set `icon_links` but it doesn't sound like the them is staying quite minimalistic: https://github.com/pradyunsg/furo/discussions/176